modules/twitch-notifications/configs/streamers.json

```
@@ -79,6 +79,26 @@
       "type": "string",
       "description-en": "Streamer where a notification should send when they start streaming",
       "description-de": "Steamer, bei denen eine Benachrichtigung gesendet werden soll, wenn sie anfangen, zu streamen"
+    },
+    {
+      "field_name": "id",
+      "humanname-en": "Discord-User ID",
+      "humanname-de": "Discord-Benutzer ID",
+      "default": "",
+      "type": "string",
```

If you want, we can merge #76 before this so the new type is supported.
```suggestion
      "type": "userID",
```

---

Yeah, would be great

---

Merged :+1: